skyportal/handlers/api/observingrun.py

```diff
@@ -227,7 +227,7 @@ def put(self, run_id):
         self.push_all(action="skyportal/FETCH_OBSERVING_RUNS")
         return self.success()
 
-    @permissions(["Upload data"])
+    @auth_or_token
```

Comment:
Why is this one not with a "permissions"?

Reply:
The permissions are done in the handler (we also do this for others)

Reply:
OK just checking.